### PR TITLE
cli: Shows message for no meshes

### PR DESCRIPTION
**Pull request by mudit-01**
Shows message if no mesh exits in the cluster fixes https://github.com/openservicemesh/osm/issues/4697

Signed-off-by: mudit singh <mudit.singh@india.nec.com>

**Description**:

**Testing done**: yes

**Affected area**:
| Functional Area            |     |
| -------------------------- | --- |
| New Functionality          | [ ] |
| CI System                  | [ ] |
| CLI Tool                   | [ x] |
| Certificate Management     | [ ] |
| Control Plane              | [ ] |
| Demo                       | [ ] |
| Documentation              | [ ] |
| Egress                     | [ ] |
| Ingress                    | [ ] |
| Install                    | [ ] |
| Networking                 | [ ] |
| Observability              | [ ] |
| Performance                | [ ] |
| SMI Policy                 | [ ] |
| Security                   | [ ] |
| Sidecar Injection          | [ ] |
| Tests                      | [ ] |
| Upgrade                    | [ ] |
| Other                      | [ ] |

Please answer the following questions with yes/no.

1. Does this change contain code from or inspired by another project?
    -   Did you notify the maintainers and provide attribution?

2. Is this a breaking change?

3. Has documentation corresponding to this change been updated in the [osm-docs](https://github.com/openservicemesh/osm-docs) repo (if applicable)?

### CODE REVIEW

**Comment by jaellio** [comment]
Hi @mudit-01! Thank you for this fix! Please let us know if need any additional support completing this PR!

**Review by schristoff — COMMENTED** [review]
Thank you so much for this contribution!

File: cmd/cli/version_test.go
Comment (mudit-01) [reply cmd/cli/version_test.go:220]: sure I will add 👍🏽

File: cmd/cli/version.go
Comment (schristoff) [reply cmd/cli/version.go:174]: Yeah that sounds nice :)

**Review by jaellio — APPROVED** [review]

**Comment by jaellio** [comment]
Thanks @mudit-01!

**Comment by mudit-01** [comment]
Thank you @jaellio for the review.

**Review by shalier — APPROVED** [review]

**Comment by mudit-01** [comment]
Thanks you @shalier for the review

**Comment by jaellio** [comment]
@shashankram @nojnhuh @trstringer could I get another review on this?

**Comment by mudit-01** [comment]
@shashankram @trstringer @nojnhuh waiting for your review.

**Comment by mudit-01** [comment]
@shashankram @trstringer @nojnhuh @snehachhabria @draychev could I get another review on this?

**Review by shashankram — APPROVED** [review]

**Pull request merged**
